lib/sles4sap_publiccloud.pm

```diff
@@ -215,6 +218,97 @@ sub stop_hana {
     }
 }
 
+=head2 start_hana
+    start_hana([timeout => 60]);
+
+    Start HANA DB using "HDB start" command
+=cut
+
+sub start_hana {
+    my ($self) = @_;
+    $self->run_cmd(cmd => "HDB start", runas => get_var("SAP_SIDADM"));
```

Comment:
Does `SAP_SIDADM` needs to be configurable? I think that it is hardcoded somewhere in qe-sap-deployment Ansible code. So we need to take care to have a job setting to match to an Ansible hardcoded value in qe-sap-deployment. Do we need to make it configurable in qe-sap-deployment (like a conf.yaml variable) too?

Reply:
In this case "SAP_SIDADM" is defined internally inside openQA deployment module and is required for the testcode, not the deployment project. 
https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/7dd0fc351b578477941cc11b0b656fa25caf9410/tests/sles4sap/publiccloud/qesap_terraform.pm#L98

But I should change it to get_required_var just in case if the deployment was done in some other way. 
Or just change it to "lc(get_var("INSTANCE_SID") . "adm")"